src/hello-world.md

```diff
@@ -262,7 +262,7 @@ ENTRY(start)
 ```
 
 This sets the ‘entry point’ for this executable. In our case, we called our
-entry point by the name people use: `start`. Remember? In `boot.s`? Same
```

Comment:
Ah I didn't notice this had already been fixed here: https://github.com/intermezzOS/book/pull/40